service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -93,7 +93,7 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
                 "AND \n" +
                 "   adminItems.isactive = 1 \n" +
                 "AND \n" +
-                "   formItem.isactive = 1";
+                "   (formItem.isactive = 1 OR formItem.isactive IS NULL)";
```

Comment:
the `isactive IS NULL` is because of the `LEFT JOIN` would have `NULL`s for all the columns in testformitem